README.md

```diff
@@ -51,14 +51,28 @@ To stop the environment use the same script, just make sure to pass the `local`
 $ ./bin/stop-environment.sh
 ```
   
-Once the containers are up, we can then create caseworker user to login into CCD. 
+Once the containers are up, we can then create a caseworker user to login into CCD. 
 
-By default `caseworker-sscs` role is assigned to the user.
-If you want to change the role pass in appropriate role while executing the script. User role should exists in Idam.
+The required user should have the `caseworker-bulkscan` role. This matches the CCD definition loaded at the next stage.
 
-  ```bash
-   $ ./bin/create-case-worker.sh
-  ```
+Before running the `docker/ccd-definition-import/scripts/caseworker-bulkscan-idam.sql` sql script you need to edit it and
```

Comment:
This should be automated with a container that adds the user, 
Password should not be different for local environment as it adds unneeded complexity

Reply:
I discussed this with @nitinprabhu and we got to the conclusion that better safe than sorry and so the password is available in vault.

Reply:
this is silly complexity and absolutely not needed for local development

Reply:
For local it should be fine as Tim suggested but for AAT and DEMO password needs to be in vault ?

Reply:
To run functional tests we need sql script to be executed on aat and demo

Reply:
Yes, see readme file, both of you :-)